<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OMR Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>OMR Result</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
              </property>
              <property name="text">
               <string>Solmization Convertion</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
              </property>
              <property name="text">
               <string>Solmizations</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QPushButton" name="pushButton_convert">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Convert</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_openImg">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>OpenImg</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QTextBrowser" name="textBrowser_solmizations"/>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="pushButton_clear">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
              </property>
              <property name="text">
               <string>Transformation</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_select">
                <property name="text">
                 <string>Select</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_done">
                <property name="text">
                 <string>Done</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
              </property>
              <property name="text">
               <string>Open Image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_restore">
              <property name="text">
               <string>Restore</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QGraphicsView" name="graphicsView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>10000</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QGraphicsView" name="graphicsView_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>10000</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
            </property>
            <property name="text">
             <string>Image with Note Point</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>OMR Process</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>Horizontal Projection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>Line Removed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>StaffLine Recognition</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGraphicsView" name="graphicsView_3">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="foregroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGraphicsView" name="graphicsView_4"/>
        </item>
        <item row="1" column="2">
         <widget class="QGraphicsView" name="graphicsView_5"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>Template Matching</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>Template matching Binary</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 221, 221); border-color: rgb(8, 8, 8);
font: italic 9pt &quot;Franklin Gothic Demi&quot;;</string>
          </property>
          <property name="text">
           <string>Image With Note Point</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGraphicsView" name="graphicsView_6"/>
        </item>
        <item row="3" column="1">
         <widget class="QGraphicsView" name="graphicsView_7"/>
        </item>
        <item row="3" column="2">
         <widget class="QGraphicsView" name="graphicsView_8"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
